react/features/notifications/functions.js

```diff
@@ -11,11 +9,9 @@ declare var interfaceConfig: Object;
  * @param {Object|Function} stateful - The redux store state.
  * @returns {boolean}
  */
-export function areThereNotifications(stateful: Object | Function) {
-    const state = toState(stateful);
-    const { enabled, notifications } = state['features/notifications'];
+export function areThereNotifications() {
 
-    return enabled && notifications.length > 0;
+    return false;
```

Comment:
What does this deactivation of notifications?

Reply:
As you wrote, this deactivates the notifications (all of them).